pkg/crc/validation/validation.go

```diff
@@ -67,10 +67,11 @@ func ValidateBundlePath(bundlePath string, preset crcpreset.Preset) error {
 
 func ValidateBundle(bundlePath string, preset crcpreset.Preset) error {
 	bundleName := filepath.Base(bundlePath)
-	_, err := bundle.Get(bundleName)
+	bundleMetadata, err := bundle.Get(bundleName)
 	if err != nil {
 		return ValidateBundlePath(bundlePath, preset)
 	}
+	bundleMismatchWarning(bundleMetadata.GetBundleName(), preset)
```

Comment:
Hmm or is the `userProvidedBundle = bundle.GetBundleNameWithExtension(userProvidedBundle)` call needed by this commit and only this one?

Reply:
Answered elsewhere, it's not related to this commit.